File: pixo-c#/DalSql/models/Customization.cs
using System;
using System.Collections.Generic;

namespace DalSql.models;

public partial class Customization
{
    public int Id { get; set; }

    public int? ProductId { get; set; }

    public string? TextToPrint { get; set; }

    public string? ColorText { get; set; }

    public int? SizeText { get; set; }

    public string? FontName { get; set; }

    public virtual ICollection<OrderItem> OrderItems { get; set; } = new List<OrderItem>();

    public virtual Product? Product { get; set; }
}

File: projectAngular/src/app/models/customization.ts
export class Customization {
  id: number;             // מזהה ההתאמה
  productId: number;      // מזהה המוצר
  textToPrint: string;    // טקסט להדפסה
  colorText: string;      // צבע טקסט
  fontName: string;       // שם הפונט
  sizeText: number;       // גודל הטקסט

  constructor(
    id: number,
    productId: number,
    textToPrint: string,
    colorText: string,
    fontName: string,
    sizeText: number
  ) {
    this.id = id;
    this.productId = productId;
    this.textToPrint = textToPrint;
    this.colorText = colorText;
    this.fontName = fontName;
    this.sizeText = sizeText;
  }
}

